app/views/auth/signup.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="col-md-offset-4 col-md-4">
+  <form class="form-signin" method="POST" action="./signup">
+          <h2 class="form-signin-heading">New Account</h2>
+
+
+
+          <label for="inputUsername" class="sr-only">Username:</label>
+          <input id="inputUsername" class="form-control field-margin" placeholder="Username" name="user[username]" required >
+
+          <label  class="sr-only">Email address</label>
```

Comment:
This might be more readable if you'd used form_tag rather than an explicit form. Is this copied from a sinatra proj?

Not using the form helpers where appropriate would be frowned upon in a rails shop.